examples/compiled/layer_cumulative_histogram.vg.json

```diff
@@ -33,13 +33,10 @@
         },
         {"type": "filter", "expr": "datum.bin_IMDB_Rating !== null"},
         {
-          "type": "window",
-          "params": [null],
+          "type": "joinaggregate",
           "as": ["cumulative_count"],
           "ops": ["sum"],
-          "fields": ["count"],
-          "sort": {"field": ["bin_IMDB_Rating"], "order": ["ascending"]},
-          "frame": [null, 0]
+          "fields": ["count"]
```

Comment:
Cumulative frequency should not use join aggregate.  

Window with frame = `[null, 0]` makes more sense. (See SVG)